pkg/api/v1/conditions/routes/handlers.go

```diff
@@ -366,11 +348,62 @@ func (r *Routes) firmwareInstall(c *gin.Context) (int, *v1types.ServerResponse)
 	return http.StatusOK, &v1types.ServerResponse{
 		Message: "firmware install scheduled",
 		Records: &v1types.ConditionsResponse{
-			ServerID: serverID,
-			State:    rctypes.Pending,
-			Conditions: []*rctypes.Condition{
-				fwCondition,
-				invCondition,
+			ServerID:   serverID,
+			State:      rctypes.Pending,
+			Conditions: serverConditions.Conditions,
+		},
+	}
+}
+
+func (r *Routes) firmwareInstallComposite(serverID uuid.UUID, fwtp rctypes.FirmwareInstallTaskParameters) *rctypes.ServerConditions {
+	createTime := time.Now()
+	return &rctypes.ServerConditions{
+		ServerID: serverID,
+		Conditions: []*rctypes.Condition{
+			{
+				Kind:    rctypes.BrokerAcquireServer,
+				Version: rctypes.ConditionStructVersion,
+				Parameters: rctypes.NewBrokerTaskParameters(
+					serverID,
+					rctypes.AcquireServer,
+					rctypes.PurposeFirmwareInstall,
+					"Marked for firmware install",
+				).MustMarshal(),
+				State:     rctypes.Pending,
+				CreatedAt: createTime,
+			},
+			{
+				Kind:       rctypes.FirmwareInstallInband,
+				Version:    rctypes.ConditionStructVersion,
```

Comment:
so we'll unconditionally queue an inband install? Will that be the default?

Reply:
Since we insist all changes trickle into main first, this  cannot end up in main as is,
I'll comment out the additional `acquire`, `release`, `inband` conditions.

Separate changes are planned for Conditions API to compose the firmware set itself,
that would then mean the firmware set composition will dictate if inband is required or not.
Although I had not planned for introducing this change as yet, but now it seems to be more necessary than not.